components/fieldgroups/weather/forecasted-weather.fieldgroup.json

```diff
@@ -0,0 +1,343 @@
+                                    "title": "Day 10 Forecast"
+                                },
+                                "xdm:day14Forecast": {
+                                    "$ref": "#/definitions/longRange",
+                                    "description": "Weather forecast for fourteen days.",
+                                    "title": "Day 14 Forecast"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "shortRange": {
+            "properties": {
+                "xdm:qpf": {
+                    "title": "qpf",
+                    "description": "The forecasted measurable precipitation (liquid or liquid equivalent) during the 12 or 24 hour period. Measured in millimeters.",
+                    "$ref": "https://ns.adobe.com/xdm/mixins/current-weather#/definitions/precipitation"
+                },
+                "xdm:calendarDayTemperatureMax": {
+                    "title": "calendarDayTemperatureMax",
+                    "description": "The midnight to midnight daily maximum temperature for the given day.",
+                    "$ref": "https://ns.adobe.com/xdm/mixins/current-weather#/definitions/temperature"
+                },
+                "xdm:calendarDayTemperatureMin": {
+                    "title": "calendarDayTemperatureMin",
+                    "description": "The midnight to midnight daily minimum temperature for the given day.",
+                    "$ref": "https://ns.adobe.com/xdm/mixins/current-weather#/definitions/temperature"
+                },
+                "xdm:day": {
+                    "title": "day",
+                    "description": "Weather information for a daytime period",
+                    "$ref": "#/definitions/dayPart"
+                },
+                "xdm:night": {
+                    "title": "night",
+                    "description": "Weather information for a nighttime period",
+                    "$ref": "#/definitions/dayPart"
+                },
+                "xdm:qpfSnow": {
+                    "title": "qpfSnow",
+                    "description": "The forecasted measurable precipitation as snow during the 12 or 24 hour forecast period. Measured in centimeters.",
+                    "$ref": "https://ns.adobe.com/xdm/mixins/current-weather#/definitions/snowfall"
+                }
+            },
+            "type": "object"
+        },
+        "longRange": {
+            "properties": {
+                "xdm:qpf": {
+                    "title": "qpf",
+                    "description": "The forecasted measurable precipitation (liquid or liquid equivalent) during the 24 hour period.",
+                    "$ref": "https://ns.adobe.com/xdm/mixins/current-weather#/definitions/precipitation"
+                },
+                "xdm:calendarDayTemperatureMax": {
+                    "title": "calendarDayTemperatureMax",
```

Comment:
Let's make the title a proper sentence. "Calendar Day Temperature Max". Please apply this in every other title